app/src/main/java/com/example/playlistmaker/feature_medialibrary/presentation/MediaLibraryFragment.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.playlistmaker.feature_medialibrary.presentation
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.playlistmaker.databinding.FragmentMediaLibraryBinding
+import com.google.android.material.tabs.TabLayoutMediator
+
+class MediaLibraryFragment : Fragment() {
+
+    private var _binding: FragmentMediaLibraryBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentMediaLibraryBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupViewPager()
+    }
+
+    private fun setupViewPager() {
+        val adapter = MediaLibraryPagerAdapter(requireActivity())
+        binding.viewPager.adapter = adapter
+
+        TabLayoutMediator(binding.tabLayout, binding.viewPager) { tab, position ->
+            tab.text = when (position) {
+                0 -> getString(com.example.playlistmaker.R.string.favorite_tracks)
+                1 -> getString(com.example.playlistmaker.R.string.playlists)
+                else -> ""
```

Comment:
Здесь можно упростить:

```suggestion
                else -> getString(com.example.playlistmaker.R.string.playlists)
```